test_data/issue-125.good.nix

```diff
@@ -15,8 +15,10 @@ let
     in
     x;
   baz =
-    let x = 3;
-    in x;
```

Comment:
This is valid syntax. I don't think we should break that.

Reply:
because it contains newline between `let` and `in`

Reply:
There should be an exception if the new line is between the `in` and the previous token.